install_config/aggregate_logging.adoc

```diff
@@ -838,7 +899,7 @@ the new `openshift_logging_es_cluster_size` value is higher than the current num
 of Elasticsearch nodes (scaled up) in the cluster.
 ====
 
-If you do not wish to reinstall, for instance because you have made
+If you do not wish to reinstall, for example, you have made
```

Comment:
I would remove this paragraph all together..  The deployer is no longer a thing and we do not recommend adding nodes other then through ansible and the openshift installer